submarine-vis/src/components/App.js

```diff
@@ -16,11 +17,39 @@ class App extends Component {
       isSafetyConditionAchieved: true,
       submarineRegion: 'yellow',
       trenchAlert: 'red',
-      probes: [96, 4, 7],
+      probes: [],
+      turns: [],
     };
   }
 
-  // TODO: have a function that talks to the server for game stats
+  componentDidMount() {
+    this.interval = setInterval(() => {
+      let turn = this.state.turns.shift()
```

Comment:
Let me get this straight - `turns` is suppose to be a queue of all the iterations of the game?

Reply:
Yes, that's correct. I'm changing it to gamestates as that might be more clear.